editions/tw5.com/tiddlers/system/$__editions_tw5.com_wikitext-template.tid

```diff
@@ -0,0 +1,13 @@
+code-body: yes
+created: 20220726132935807
+list-after: $:/core/ui/ViewTemplate/body
+modified: 20220726134215137
+tags: $:/tags/ViewTemplate
+title: $:/editions/tw5.com/wikitext-template
+
+<$list filter="[all[current]tag[WikiText]]" variable="listItem">
```

Comment:
Again this feels like a local, isolated fix to a universal issue. All we're doing here is providing a link back to one of the tags on the current tiddler. That's already possible using the tag pill. Randomly providing another means to do the same thing just makes it look like we don't trust the tagging mechanism as a means of user navigation.

Reply:
From the feedback we get from new users. It isn't enough, to have "hidden information" behind the tag-pill. Because new users don't know about it. 

If I search for "Formatting" in the sidebar search I may find: "Formatting in Wikitext" and open it. ... My expectation is, that I do get all the info. ... But the tiddler only contains one part of the info. ... That's why I think it makes sense to tell the users, that there is more.

Reply:
That's all reasonable logic, but if it is true then it would actually be an argument for adding breadcrumbs to each docs tiddler, not for making a random isolated improvement for a single tag.

Reply:
OK. What about a "tip" eg: `<<.tip "The WikiText tag-pill at the top of the tiddler will show you all wikitext rules">> `

Reply:
eg: IMO It's a call to action to "use the tagging system" and if they click the link the effect is similar

![grafik](https://user-images.githubusercontent.com/374655/181185814-24a2bd57-4891-41d9-b46e-9cf3e570e178.png)